### PR TITLE
Test Example

### PR DESCRIPTION
**Pull request by noah4546**

### CODE REVIEW

File: src/main/java/frc/teleop/TeleopDriver.java [hunk 1]
```diff
@@ -30,6 +30,8 @@ public static TeleopDriver getInstance() {
     private TeleopDriver() {
         this.drive = Drive.getInstance();
         this.intake = Pickup.getInstance();
+
+        // HI MY NAME IS NOAH!
```
Comment (noah4546) [inline src/main/java/frc/teleop/TeleopDriver.java:34]: Why this comment? No need please remove.

Comment (noah4546) [reply src/main/java/frc/teleop/TeleopDriver.java:34]: Ok Fixed

Comment (DorianMotie) [reply src/main/java/frc/teleop/TeleopDriver.java:34]: cool

**Review by DorianMotie — APPROVED** [review]
it's cool.

**Pull request closed without merging**